src/test/util/uvu-timeout.ts

```diff
@@ -6,7 +6,8 @@
 
 import type * as uvu from 'uvu';
 
-const DEFAULT_TIMEOUT = Number(process.env.TEST_TIMEOUT ?? 30_000);
+const TIMEOUT_IF_UNSET = process.env.CI ? 30_000 : 10_000;
```

Comment:
+1 I've been wanting this.

I'd actually maybe go even lower for local. I usually use 3000 which seems to work just fine on my computer. Maybe 5000 to be safe?